docs/source/contents/4_draft_mapping.rst

```diff
@@ -1777,10 +1777,644 @@ Decision
+of the distinct Xpaths are given below:
+
+`acwiley:280 as an example of mods:subject[@valueURI]/mods:topic <https://digital.lib.utk.edu/collections/islandora/object/acwiley%3A280/datastream/MODS/view>`_
+
+.. code-block:: xml
+
+    <subject authority="lcsh" valueURI="http://id.loc.gov/authorities/subjects/sh85147554">
+        <topic>Women in art</topic>
+    </subject>
+    <subject authority="lcsh" valueURI="http://id.loc.gov/authorities/subjects/sh85147447">
+        <topic>Women artists</topic>
+    </subject>
+    <subject authority="tgm" valueURI="http://id.loc.gov/vocabulary/graphicMaterials/tgm008085">
+        <topic>Portraits</topic>
+    </subject>
+
+.. code-block:: turtle
+
+    @prefix dcterms: <http://purl.org/dc/terms/> .
+
+    <https://example.org/objects/1> dcterms:subject "http://id.loc.gov/authorities/subjects/sh85147554" ;
+        dcterms:subject "http://id.loc.gov/authorities/subjects/sh85147447" ;
+        dcterms:subject "http://id.loc.gov/vocabulary/graphicMaterials/tgm008085" .
+
+`cdf:5384 as an example of mods:subject
+/mods:topic[@valueURI] <https://digital.lib.utk.edu/collections/islandora/object/cdf%3A5384/datastream/MODS/view>`_
+
+.. code-block:: xml
+
+    <subject>
+        <topic valueURI="http://id.loc.gov/authorities/subjects/sh85023396">Child welfare</topic>
+    </subject>
+
+.. code-block:: turtle
+
+    @prefix dcterms: <http://purl.org/dc/terms/> .
+
+    <https://example.org/objects/1> dcterms:subject "http://id.loc.gov/authorities/subjects/sh85023396" .
+
+`wwiioh:2451 as an example of mods:subject[@valueURI]/mods:name/mods:namePart <https://digital.lib.utk.edu/collections/islandora/object/wwiioh%3A2451/datastream/MODS/view>`_.
+
+.. code-block:: xml
+
+    <subject authority="naf" valueURI="http://id.loc.gov/authorities/names/n85185770">
+        <name>
+            <namePart>United States. Army. Medical Corps</namePart>
+        </name>
+    </subject>
+
+.. code-block:: turtle
+
+    @prefix dcterms: <http://purl.org/dc/terms/> .
+
+    <https://example.org/objects/1> dcterms:subject "http://id.loc.gov/authorities/names/n85185770" .
+
+`helser:24792 as an example of mods:subject/mods:name[@valueURI] <https://digital.lib.utk.edu/collections/islandora/object/hesler%3A24792/datastream/MODS/view>`_.
+
+.. code-block:: xml
+
+    <subject>
+        <name authority="naf" valueURI="http://id.loc.gov/authorities/names/n87116131">
+            <namePart>Atkinson, George Francis, 1854-1918</namePart>
+        </name>
+    </subject>
+    <subject>
+        <name authority="naf" valueURI="http://id.loc.gov/authorities/names/n88144876">
+            <namePart>Arthur, Joseph Charles, 1850-1942</namePart>
+        </name>
+    </subject>
+
+.. code-block:: turtle
+
+    @prefix dcterms: <http://purl.org/dc/terms/> .
+
+    <https://example.org/objects/1> dcterms:subject "http://id.loc.gov/authorities/names/n88144876" ;
+        dcterms:subject "http://id.loc.gov/authorities/names/n87116131" .
```

Comment:
Same question regarding double quotes vs angle brackets.

Reply:
:100: :clap:

Reply:
Thank you - I addressed this in the most recent commit.